auto_round/auto_quantizer.py

```diff
@@ -41,7 +41,7 @@
 from transformers.quantizers import AutoQuantizationConfig, HfQuantizer
 from transformers.quantizers.auto import AUTO_QUANTIZER_MAPPING
 from transformers.utils.quantization_config import AwqConfig, GPTQConfig, QuantizationConfigMixin, QuantizationMethod
-
+from transformers.quantizers.auto import AUTO_QUANTIZATION_CONFIG_MAPPING
```

Comment:
do all transformers>4.38 have this one?

Reply:
yes, I checked previous versions
![image](https://github.com/user-attachments/assets/dcb511d6-62e4-43fc-bd76-fd546d53d02e)